xbmc/utils/LangCodeExpander.cpp

```diff
@@ -680,6 +680,7 @@ const std::array<struct LCENTRY, 185> g_iso639_1 = {{
     {MAKECODE('\0', '\0', 'p', 'l'), "Polish"},
     {MAKECODE('\0', '\0', 'p', 's'), "Pashto, Pushto"},
     {MAKECODE('\0', '\0', 'p', 't'), "Portuguese"},
+    {MAKECODE('\0', '\0', 'p', 'b'), "Portuguese (Brazil)"},
```

Comment:
Can we add a comment before this line. Something like:

`// unofficial language code for Brazilian Portuguese`

Reply:
sure